Automattic/woocommerce-payments
# Replace "payouts" to "deposits"

Fixes https://github.com/Automattic/woocommerce-payments/issues/5689

#### Changes proposed in this Pull Request

Replace all user-facing wording from "payouts" to "deposits".
I searched all "payout" in the code base and change them to "deposits" if they are user-facing.

There are several places that change:

1. Payments > Settings

![Screenshot 2023-03-16 at 21 04 52](https://user-images.githubusercontent.com/73803630/225642265-2f0581f2-6faf-46a8-8460-2a59c6819d2b.png)

2. Payments menu when store is in disconnect state.

![Screenshot 2023-03-16 at 21 09 09](https://user-images.githubusercontent.com/73803630/225643331-c6e1fe84-579a-4a5b-ac03-09ecfb7619df.png)


#### Testing instructions

* Run `npm run build:client`.
* When store is connected to a WCpay account, open Payments > Settings. Inside the Deposit Schedule box, with base branch, you will see 'payouts' but with PR branch, it'd be 'deposits'.
* Disconnect store so it's not connected to a WCPay account. You can do so using WCPay Dev Tool by checking the 'Force onboarding' and 'Force the plugin to act as disconnected from WCPay'. The box at the bottom has 'payouts' in base branch but with PR branch it will use the word 'deposits'.

-------------------


**Post merge**
